src/stactools/landsat/mtl_metadata.py

```diff
@@ -68,10 +68,8 @@ def processing_level(self) -> str:
     def epsg(self) -> int:
         utm_zone = self._root.find_text('PROJECTION_ATTRIBUTES/UTM_ZONE')
         if utm_zone:
-            bbox = self.bbox
             utm_zone = self._get_text('PROJECTION_ATTRIBUTES/UTM_ZONE')
-            center_lat = (bbox[1] + bbox[3]) / 2.0
-            return int(f"{326 if center_lat > 0 else 327}{utm_zone}")
+            return int(f"326{utm_zone}")
```

Comment:
Can you throw a comment in here about how the transform, as provided by USGS, requires the UTM zone to always be north, even if the data are in the southern hemisphere?

Reply:
Yep. Done.